app/frontend/js/modals/updateExtractionDefinitionModal.js

```diff
@@ -0,0 +1,73 @@
+import { Modal } from "bootstrap";
+import { each } from "lodash";
+
+const updateExtractionDefinitionModal = document.getElementById(
+  "update-extraction-definition-modal"
+);
+
+if (updateExtractionDefinitionModal) {
```

Comment:
Same with this file, could it be broken up a bit?

Reply:
I think this one is OK for now, it's not as crazy as the other one

Reply:
If you find breaking up the other one into steps works, might still be good to do the same here.